tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -531,9 +553,12 @@ compressed_rel_setup_reltarget(RelOptInfo *compressed_rel, CompressionInfo *info
 
 			/* skip vars that aren't from the uncompressed chunk */
 			if (chunk_var->varno != info->chunk_rel->relid)
+			{
 				continue;
+			}
 
-			/* if there's a system column or whole-row reference, add a whole-
+			/*
+			 * If there's a system column or whole-row reference, add a whole-
```

Comment:
That's probably a remnant of when I added something else to this branch. That's why I don't like skipping braces for single-statement blocks, makes it harder/error-prone to modify. I can keep the old formatting here.

Reply:
Although given that I'm touching a lot of code in this function, it can be a good time to fix style of the comments and so on...

Reply:
It's not critical to not change comments or do other non-functional changes. Sometimes it is good to do it, but if we do it all the time, it will be tricky to trace down bugs and other issues later.